instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/logging/slf4j/Slf4jMdcAspect.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.instrumentation.spring.autoconfigure.logging.slf4j;
+
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.SpanContext;
+import io.opentelemetry.trace.TracingContextUtils;
+import org.aspectj.lang.annotation.After;
+import org.aspectj.lang.annotation.Aspect;
+import org.slf4j.MDC;
+
+@Aspect
+public final class Slf4jMdcAspect {
+  private static final String SPAN_ID = "spanId";
```

Comment:
maybe need to qualify it w/ the `ot`? or whatever header that ot actually use. Unless if you use `x-b3` propagation, which can prob use the same header key as x-b3?

Reply:
I removed aspects from this project instead I'm using this project to add ids to log4j's mapped diagnostic context. 

I'm thinking about moving this out of the spring-starter and creating a log4j-starter